<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Results ecm33davidjm</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<style type="text/css">
<!--
.Estilo4 {font-family: Arial, Helvetica, sans-serif}
body {
	background-color: #CCCCCC;
}
.Estilo5 {font-size: 10px}
.Estilo6 {
	font-family: Arial, Helvetica, sans-serif;
	font-size: 14px;
	font-weight: bold;
}
.Estilo7 {
	color: #990000;
	font-size: 10px;
	font-family: Arial, Helvetica, sans-serif;
}
.Estilo8 {font-family: Arial, Helvetica, sans-serif; font-size: 10px; }
.Estilo9 {font-size: 12px}
.Estilo11 {
	font-family: Arial, Helvetica, sans-serif;
	color: #990000;
	font-weight: bold;
	font-size: 24px;
}
.Estilo13 {font-family: Arial, Helvetica, sans-serif; font-size: 14px; }
.Estilo14 {font-size: 9px}
.Estilo15 {color: #990000}
.Estilo16 {color: #000000}
.Estilo17 {color: #009966}
.Estilo20 {font-family: "Courier New", Courier, mono}
.Estilo23 {font-family: Arial, Helvetica, sans-serif; font-size: 12px; }
.Estilo24 {color: #996633}
.Estilo25 {
	font-family: Arial, Helvetica, sans-serif;
	font-size: 10px;
	color: #3300FF;
	font-weight: bold;
}
.Estilo26 {
	font-family: Arial, Helvetica, sans-serif;
	font-size: 10px;
	color: #3399FF;
	font-weight: bold;
}
.Estilo27 {
	font-size: 10;
	font-weight: bold;
}
.Estilo29 {font-family: Arial, Helvetica, sans-serif; font-size: 9px; }
.Estilo30 {font-size: 14px}
.Estilo31 {font-size: 16px}
.Estilo32 {color: #666600}
.Estilo36 {font-size: 12px; font-weight: bold; }
.Estilo37 {font-size: 12}
.Estilo38 {font-size: 12; font-weight: bold; }
-->
</style>
</head>

<body>
<div align="center">
  <p><a name="up"></a></p>
  <p class="Estilo6">&quot;Caracterizaci&oacute;n de la <em>Familia de Ecm33 </em> como etapa<br>
  en la predicci&oacute;n de la estructura y funci&oacute;n de Ecm33&quot; </p>
  <p align="center" class="Estilo11">Resultados</p>
  <table width="500" border="0" cellspacing="0" cellpadding="0">
    <tr>
      <td width="278" height="52" align="left" valign="top"><span class="Estilo13">&gt; Alineamientos m&uacute;ltiples</span></td>
      <td width="222" align="left" valign="top"><p class="Estilo8">-<a href="priv/consensus08ecmgroup.pdf">Familia Ecm33</a><br>
        -<a href="priv/consensus07groupecm.pdf" target="_blank" class="Estilo15">Regi&oacute;n N-terminal</a><br>
      -<a href="priv/consensus306ecmgroup.pdf" target="_blank" class="Estilo15">Regi&oacute;n C-terminal</a></p>      </td>
    </tr>
    <tr>
      <td height="28" align="left" valign="top"><span class="Estilo13">&gt; Resultados b&uacute;squedas HMMER</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><div align="right" class="Estilo9"><span class="Estilo4">- Familia Ecm33:&nbsp;&nbsp;&nbsp; </span></div></td>
      <td align="left" valign="top"><p class="Estilo8">-- <a href="priv/08nrncbiResults.html" target="_blank" class="Estilo15">Non Redundant </a><br>
- <a href="priv/08pdbncbiResults.html" target="_blank" class="Estilo15">Pdb</a><br>
<em>- S. cerevisiae<br>
- Candidadb<br>
- <a href="priv/08pompepResults.html" target="_blank" class="Estilo15">S.pombe</a></em> <br>
      </p>      </td>
    </tr>
    <tr>
      <td align="left" valign="top"><div align="right" class="Estilo9"><span class="Estilo4">- N-terminal Familia Ecm33: &nbsp;&nbsp;</span></div></td>
      <td align="left" valign="top"><p class="Estilo8">- <a href="priv/210152.hmm.nrdb.Results.html" target="_blank" class="Estilo15">Non Redundant</a> <br>
        - <a href="priv/07pdbncbiResults.html" target="_blank" class="Estilo15">Pdb</a><br>
        <em>- <a href="priv/07yeastResults.html" target="_blank" class="Estilo15">S. cerevisiae</a><br>
        - <a href="priv/07candidadbResults.html" target="_blank" class="Estilo15">Candidadb</a><br>
        - <a href="priv/07pompeResults.html" target="_blank" class="Estilo16">S<span class="Estilo15">.pombe</span></a></em> </p>      </td>
    </tr>
    <tr>
      <td height="75" align="left" valign="top"><div align="right" class="Estilo9"><span class="Estilo4">- C-terminal Familia Ecm33:&nbsp;&nbsp;&nbsp; </span></div></td>
      <td align="left" valign="top"><span class="Estilo8">- <a href="priv/06nrncbiResults.html" target="_blank" class="Estilo15">Non Redundant</a> <br>
- <a href="priv/06pdbncbiResults.html" target="_blank" class="Estilo15">Pdb</a><br>
<em>- <a href="priv/06yeastResults.html" target="_blank" class="Estilo15">S. cerevisiae</a><br>
- <a href="priv/06candidaResults.html" target="_blank" class="Estilo15">Candidadb</a><br>
- <a href="priv/06pompepResults.html" target="_blank" class="Estilo15">S.pombe</a></em> </span></td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; <a href="#GPI" class="Estilo17">Predicci&oacute;n de P&eacute;ptido Se&ntilde;al y GPI</a></span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SaEcm33</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; CaEcm33p</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SaPst1</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; CaPst1</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SaSps2</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SaSps22</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SpMeu10</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; EgSps22</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; CaSps22</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; SpYin3|YIN3_SCHPO</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; FFAS 003 Fold recognition analyses</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
    <tr>
      <td align="left" valign="top"><span class="Estilo13">&gt; Robetta Resultados</span></td>
      <td align="left" valign="top">&nbsp;</td>
    </tr>
  </table>
  <table width="500" border="0" cellspacing="0" cellpadding="0">
    <tr>
      <td><p>&nbsp;</p>
        <p><span class="Estilo13"><a name="GPI"><strong>Predicci</strong></a><strong>&oacute;n de P&eacute;ptido Se&ntilde;al y GPI </strong></span><span class="Estilo4"><strong></strong></span></p>        <table width="482" cellpadding="0" cellspacing="0">
          <tr>
            <td width="75"><p align="center"><strong><span class="Estilo4"><span class="Estilo5"></span></span></strong></p></td>
            <td width="91"><p align="center" class="Estilo8"><strong>Signal IP </strong></p></td>
            <td width="81"><p align="center" class="Estilo8"><strong>Psort II </strong></p></td>
            <td width="79"><p align="center" class="Estilo8"><strong>GPI </strong></p></td>
            <td width="80"><p align="center" class="Estilo8"><strong>DGPI </strong></p></td>
            <td width="53"><p class="Estilo8"><strong>Dem <br>
        Gpi </strong></p></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>SaEcm33 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (21-22) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (19-20) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (445) </p></td>
            <td width="80" valign="top"><p class="Estilo8">SI (445) </p></td>
            <td width="53" valign="top"><p class="Estilo8">407* </p></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>CaEcm33 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (18-19) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (1-18) </p></td>
            <td width="79" valign="top"><p class="Estilo8">Si (399) </p></td>
            <td width="80" valign="top"><p class="Estilo8">Si (394) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>SaPst1 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (19-20) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (24-25) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (419) </p></td>
            <td width="80" valign="top"><p class="Estilo8">No (419) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>CaPst1 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (20-21) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (23-24) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (384) </p></td>
            <td width="80" valign="top"><p class="Estilo8">Si (385) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>SaSps2 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">No (23-24) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (56-57) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (475) </p></td>
            <td width="80" valign="top"><p class="Estilo8">No (475) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>SaSps22 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (25-26) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (1-25) </p></td>
            <td width="79" valign="top"><p class="Estilo8">Si (440) </p></td>
            <td width="80" valign="top"><p class="Estilo8">Si (440) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>CaSps22 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">No (28-29) </p></td>
            <td width="81" valign="top"><p class="Estilo8">No </p></td>
            <td width="79" valign="top"><p class="Estilo8">Si (452) </p></td>
            <td width="80" valign="top"><p class="Estilo8">No (440) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>SpMeu10 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (29-30) </p></td>
            <td width="81" valign="top"><p class="Estilo8">No (29-30) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (395) </p></td>
            <td width="80" valign="top"><p class="Estilo8">Si (389) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"><p class="Estilo8"><strong>EgSps22 </strong></p></td>
            <td width="91" valign="top"><p class="Estilo8">Si (19-20) </p></td>
            <td width="81" valign="top"><p class="Estilo8">Si (19-20) </p></td>
            <td width="79" valign="top"><p class="Estilo8">No (457) </p></td>
            <td width="80" valign="top"><p class="Estilo8">Si (455) </p></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"></td>
            <td width="91" valign="top"></td>
            <td width="81" valign="top"></td>
            <td width="79" valign="top"></td>
            <td width="80" valign="top"></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"></td>
            <td width="91" valign="top"></td>
            <td width="81" valign="top"></td>
            <td width="79" valign="top"></td>
            <td width="80" valign="top"></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td width="75" valign="top"></td>
            <td width="91" valign="top"></td>
            <td width="81" valign="top"></td>
            <td width="79" valign="top"></td>
            <td width="80" valign="top"></td>
            <td width="53" valign="top"></td>
          </tr>
          <tr>
            <td colspan="6" valign="top"><p class="Estilo8">* {Terashima, Hamada, et al. 218 28 /id} </p></td>
          </tr>
        </table>
      <p align="right"><a href="#up" class="Estilo24">Subir</a></p>
      <p><strong>&gt; <span class="Estilo4"><a name="SaEcm33" class="Estilo11">SaEcm33 </a></span></strong></p>
      <p class="Estilo6 Estilo32"><strong>Contexto gen&eacute;tico </strong></p>
      <p class="Estilo23">Ecm33 se encuentra en el Cromosoma II, bajo el siguiente contexto gen &eacute; tico: Region Displayed: 376K-411K bp </p>
      <table cellspacing="0" cellpadding="0">
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">TAT1 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5"><strong>Amino acid transport protein for valine, leucine, isoleucine, and tyrosine </strong></p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">YBR070C </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5"><strong>Protein required for cell viability </strong></p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">YBR071W </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Hypothetical ORF </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5"><strong>HSP26 </strong></p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5"><strong>Small heat shock protein with chaperone activity that is regulated by a heat induced transition from an inactive oligomeric (24-mer) complex to an active dimer; induced by heat, upon entry into stationary phase, and during sporulation </strong></p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">YBR072C-A </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Identified by fungal homology and RT-PCR </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">RDH54 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">genetic interaction with DMC1; Putative helicase similar to RAD54 </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">YBR074W </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Hypothetical ORF </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">ECM8 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Non-essential protein of unknown function </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">YBR077C </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Hypothetical ORF </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo25">ECM33 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo26">GPI-anchored protein of unknown function, has a possible role in apical bud growth; GPI-anchoring on the plasma membrane crucial to function; similar to Sps2p and Pst1p </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">RPG1 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Subunit of the core complex of translation initiation factor 3(eIF3), essential for translation; part of a subcomplex (Prt1p-Rpg1p-Nip1p) that stimulates binding of mRNA and tRNA(i)Met to ribosomes </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5"><strong>SEC18 </strong></p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5"><strong>cytoplasmic protein involved in protein transport between ER and Golgi; ATPase </strong></p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">SPT7 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">Subunit of the SAGA transcriptional regulatory complex, involved in proper assembly of the complex; also present as a C-terminally truncated form in the SLIK/SALSA transcriptional regulatory complex </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">TR(UCU)B </p></td>
          <td width="560" valign="top" class="Estilo4"></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">TD(GUC)B </p></td>
          <td width="560" valign="top" class="Estilo4"></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5"><strong>UBC4 </strong></p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5"><strong>Ubiquitin-conjugating enzyme that mediates degradation of short-lived and abnormal proteins; interacts with E3-CaM in ubiquitinating calmodulin; interacts with many SCF ubiquitin protein ligases; component of the cellular stress response </strong></p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">TEC1 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">transcription factor of the TEA/ATTS DNA-binding domain family, regulator of Ty1 expression </p></td>
        </tr>
        <tr>
          <td width="89" valign="top" class="Estilo4"><p class="Estilo5">MIS1 </p></td>
          <td width="560" valign="top" class="Estilo4"><p class="Estilo5">mitochondrial C1-tetrahydroflate synthase </p></td>
        </tr>
      </table>      <p align="left">&nbsp;</p>
      <p align="left"><span class="Estilo13"><strong>Composici</strong><strong>&oacute;n fisico-qu&iacute;</strong><strong>mica SaEcm33</strong></span></p>      
      <table cellspacing="0" cellpadding="0">
        <tr>
          <td width="168" valign="top"><p align="center" class="Estilo8"><strong>1-469 </strong></p></td>
          <td width="175" valign="top"><p align="center" class="Estilo8"><strong>21-200 </strong></p></td>
          <td width="155" valign="top"><p align="center" class="Estilo8"><strong>200-407 </strong></p></td>
        </tr>
        <tr>
          <td width="168" valign="top"><p class="Estilo8">Number of amino acids: 469<br>
            Molecular weight: 48307.9 Theoretical pI: 4.93<br>
            Amino acid composition: <br>Ala (A) 44 9.4% <br>
            Arg (R) 4 0.9% <br>
            Asn (N) 39 8.3% <br>
            Asp (D) 19 4.1% <br>
            Cys (C) 6 1.3% <br>
            Gln (Q) 16 3.4% <br>
            Glu (E) 11 2.3% <br>
            Gly (G) 30 6.4% <br>
            His (H) 0 0.0% <br>
            Ile (I) 34 7.2% <br>
            Leu (L) 45 9.6% <br>
            Lys (K) 21 4.5% <br>
            Met (M) 5 1.1% <br>
            Phe (F) 23 4.9% <br>
            Pro (P) 6 1.3% <br>
            Ser (S) 91 19.4% <br>Thr (T) 40 8.5% <br>Trp (W) 1 0.2% <br>
            Tyr (Y) 5 1.1% <br>
            Val (V) 29 6.2% <br>
            Asx (B) 0 0.0% <br>
            Glx (Z) 0 0.0% <br>
            Xaa (X) 0 0.0% </p>
              <p class="Estilo8">Negatively charged residues (Asp + Glu): 30 <br>
              Positively charged residues (Arg + Lys): 25 </p>
              <p class="Estilo8">Instability index: <br>
              The instability index (II) is computed to be 32.63 <br>
              <strong>This classifies the protein as stable. </strong></p>
              <p class="Estilo8">Aliphatic index: 93.01 </p>
              <p class="Estilo8">Grand average of hydropathicity (GRAVY): 0.185 </p></td>
          <td width="175" valign="top"><p class="Estilo8">Number of amino acids: 180 <br>
            Molecular weight: 18727.6 <br>
            Theoretical pI: 3.95 <br>
            Amino acid composition: <br>
            Ala (A) 12 6.7% <br>
            Arg (R) 1 0.6% <br>
            Asn (N) 16 8.9% <br>
            Asp (D) 11 6.1% <br>
            Cys (C) 2 1.1% <br>
            Gln (Q) 7 3.9% <br>
            Glu (E) 6 3.3% <br>
            Gly (G) 10 5.6% <br>
            His (H) 0 0.0% <br>
            Ile (I) 17 9.4% <br>
            Leu (L) 18 10.0% <br>
            Lys (K) 5 2.8% <br>
            Met (M) 2 1.1% <br>
            Phe (F) 8 4.4% <br>
            Pro (P) 2 1.1% <br>
            Ser (S) 35 19.4% <br>
            Thr (T) 19 10.6% <br>
            Trp (W) 0 0.0% <br>
            Tyr (Y) 1 0.6% <br>
            Val (V) 8 4.4% <br>
            Asx (B) 0 0.0% <br>
            Glx (Z) 0 0.0% <br>
            Xaa (X) 0 0.0% <br>
            <br>
            (Asp + Glu): 17 <br>
              (Arg + Lys): 6 <br>
              <br>
              Instability index: <br>
              The instability index (II) is computed to be 33.59 <br>
              <strong>This classifies the protein as stable. </strong></p>
            <p class="Estilo8">Aliphatic index: 95.39 </p>
              <p class="Estilo8">Grand average of hydropathicity (GRAVY): 0.097 </p></td>
          <td width="155" valign="top"><p class="Estilo8">Number of amino acids: 208 <br>
            Molecular weight: 21102.1 <br>
            Theoretical pI: 9.20 <br>
            Amino acid composition: <br>
            Ala (A) 18 8.7% <br>
            Arg (R) 2 1.0% <br>
            Asn (N) 21 10.1% <br>
            Asp (D) 8 3.8% <br>
            Cys (C) 2 1.0% <br>
            Gln (Q) 8 3.8% <br>
            Glu (E) 3 1.4% <br>
            Gly (G) 17 8.2% <br>
            His (H) 0 0.0% <br>
            Ile (I) 9 4.3% <br>
            Leu (L) 19 9.1% <br>
            Lys (K) 13 6.2% <br>
            Met (M) 0 0.0% <br>
            Phe (F) 10 4.8% <br>
            Pro (P) 1 0.5% <br>
            Ser (S) 47 22.6% <br>
            Thr (T) 15 7.2% <br>
            Trp (W) 1 0.5% <br>
            Tyr (Y) 0 0.0% <br>
            Val (V) 14 6.7% <br>
            Asx (B) 0 0.0% <br>
            Glx (Z) 0 0.0% <br>
            Xaa (X) 0 0.0% </p>
              <p class="Estilo8">(Asp + Glu): 11 <br>
              (Arg + Lys): 15 <br>
              </p>
              <p class="Estilo8">Instability index: <br>
              The instability index (II) is computed to be 34.51 <br>
              <strong>This classifies the protein as stable. </strong></p>
              <p class="Estilo8">Aliphatic index: 80.67 </p>
              <p class="Estilo8">Grand average of hydropathicity (GRAVY): -0.097 </p></td>
        </tr>
      </table>      <p align="left">&nbsp;</p>
      <p class="Estilo6"><strong>Representaci&oacute;n Gr&aacute;fica predicci&oacute;n de glicosilaci&oacute;n <span class="Estilo13"><strong>SaEcm33p</strong></span></strong></p>
      <p align="left" class="Estilo20">&gt;SaEcm33 <br>
        <span class="Estilo9">MQFKNALTATAILSASALAAA <strong>NSTT </strong>SIPSSCSIGTSATATAQADLDKISGCSTIVG <strong>NLTI </strong>TGDLGSAALASIQEIDGSLTIF <strong>NSSS </strong>LSSFSADSIKKITGDLNMQELIILTSASFGSLQE VDSINMVTLPAISTFSTDLQNANNIIVSDTTLESVEGFSTLKKVNVFNINNNRYLNSFQS <br>
        SLESVSDSLQFSSNGD <strong>NTTL </strong>AFDNLVWAN <strong>NITL </strong>RDVNSISFGSLQTV <strong>NASL </strong>GFI <strong>NNTL </strong><br>
        PS L <strong>NLTQ </strong>LSKVGQSLSIVSNDELSKAAFS <strong>NLTT </strong>VGGGFIIA <strong>NNTQ </strong>LKVIDGFNKVQTVGGAI <br>
        EVTG <strong>NFST </strong>LDLSSLKSVRGGANFDSSSS <strong>NFSC </strong>NALKKLQSNGAIQGDSFVCKNGATSTSV KLSSTSTESSKSSATSSASSSGDASNAQA <strong>NVSA </strong>SASSSSSSSKKSKGAAPELVPATSFMG VVAAVGVAYYKIKATICVSIITLISSLMISLPFLFYYETVGSSLNFICR </span> </p>
      <p align="left" class="Estilo20">&nbsp;</p>      
      <p><span class="Estilo13"><strong>Interacciones SaEcm33p </strong></span></p>
      <p class="Estilo23"><strong>2-h&iacute;bridos Levadura <br>
      </strong>YNL124W (NAF1) YBR078W (ECM33) 2 <br>
      <strong>BIND <br>
      </strong>Abf1 Fkh2 Mbp1 Ndd1 Swi4 Swi6 SLT2 Cse1 Mlp1 Mlp2 Nic96 Nup100 Nup116 Nup2 Xpo1 <br>
      <strong>DIP (Data Base of Interacting Proteins) <br>
      </strong>SGD: <a href="javascript:cop('http://db.yeastgenome.org/cgi-bin/SGD/locus.pl?locus=HRT1')">HRT1 </a> : RING finger containing subunit of Skp1-Cullin-F-box ubiquitin protein ligases (SCF); required for Gic2p, Far1p, Sic1p and Cln2p degradation; may tether Cdc34p (a ubiquitin conjugating enzyme or E2) and Cdc53p (a cullin) subunits of SCF <br>
      <strong>GRID <br>
      </strong>ECM33 was identified in association with the following 2 proteins<br>
        <em>SLT2 </em>/ YHR030C<br>
        <em>HRT1 </em>/ YOL133W</p>
      <p align="left" class="Estilo23">&nbsp;</p>
      <p align="left" class="Estilo23">&nbsp;</p>
      <p><strong>&gt; <span class="Estilo11"><a name="CaEcm33p">CaEcm33p </a></span></strong></p>      
      <p><span class="Estilo4"><span class="Estilo27">Contexto gen&eacute;tico </span></span></p>
      <p class="Estilo23"><strong>Genes in contig Chr1_Ctg10163 in the region of 'ECM33.3f' <br>
      </strong>(from 130.0 kb to 210.0 kb) (80000 bp) </p>
      <table cellspacing="0" cellpadding="0">
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">HOL1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">member of major facilitator superfamily multidrug-resistance protein subfamily 1 by homology </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">RPA1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">60S ribosomal protein </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">RPL13 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">Ribosomal protein </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">RPS16 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">ribosomal protein </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5601 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5604 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5607 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">ORC1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">Origin recognition complex protein 1 </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">TEM1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">GTP-binding protein of the RAS superfamily </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">RPS3A </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">Ribosomal protein 10 </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5353 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">RPL6 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">ribosomal protein </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5356 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">GGA1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9"><strong>Arf-binding protein: prote &iacute; na que regula a las ARF, implicada en transporte intraceluar </strong></p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5358 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5360 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9"><strong>COQ4 </strong></p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9"><strong>ubiquinone biosynthesis </strong></p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF5363 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">LAB2 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">LIPOATE BIOSYNTHESIS PROTEIN by homology </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9"><strong>ECM33.3f </strong></p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9"><strong>cell wall biogenesis, 3-prime end (by homology) </strong></p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF20024 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9"><strong>CDC12 </strong></p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9"><strong>septin (by homology) (tr &aacute; fico vesicular) </strong></p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9"><strong>BMH2 </strong></p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9"><strong>similar to Saccharomyces cerevisae Bmh2p suppressor of clathrin deficency (by homology). Bmh2 est &aacute; implicado en procesos de exocitosis y transporte vesicular. </strong></p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3709 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3708 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3707 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3704 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3701 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3698 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">similar to Saccharomyces cerevisiae Rsm24p mitochondrial ribosomal protein (S24) (by homology) </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3695 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">similar to Saccharomyces cerevisiae Ngg1p general transcriptional adaptor or co-activator (by homology) </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">MAS1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">Mitochondrial processing protease (by homology) </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3691 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF3690 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF8495 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">unknown function </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">SEC62 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">subunit of ER protein-translocation complex (by homology) </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">IPF8493 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">putative member of nontransporter group of ATP-binding cassette (ABC) superfamily (by homology) </p></td>
        </tr>
        <tr>
          <td width="96" valign="top" class="Estilo29"><p class="Estilo9">CHD1 </p></td>
          <td width="402" valign="top" class="Estilo29"><p class="Estilo9">transcriptional regulator (by homology) </p></td>
        </tr>
      </table>      <p align="left" class="Estilo29">&nbsp;</p>
      <p align="left" class="Estilo29">&nbsp;</p>
      <p><span class="Estilo4"><strong>Composici</strong><strong>&oacute;n fisico-qu&iacute;mica CaEcm33p</strong></span></p>
      <table cellspacing="0" cellpadding="0">
        <tr>
          <td width="172" valign="top"><p align="center" class="Estilo8"><strong>1-423 </strong></p></td>
          <td width="171" valign="top"><p align="center" class="Estilo8"><strong>19-204 </strong></p></td>
          <td width="155" valign="top"><p align="center" class="Estilo8"><strong>204-370 </strong></p></td>
        </tr>
        <tr>
          <td width="172" valign="top"><p class="Estilo8">Number of amino acids: 423 <br>
            Molecular weight: 43539.4 <br>
            Theoretical pI: 4.91 <br>
            Amino acid composition: <br>
            Ala (A) 41 9.7% <br>
            Arg (R) 3 0.7% <br>
            Asn (N) 34 8.0% <br>Asp (D) 20 4.7% <br>
            Cys (C) 5 1.2% <br>
            Gln (Q) 16 3.8% <br>
            Glu (E) 18 4.3% <br>
            Gly (G) 35 8.3% <br>
            His (H) 1 0.2% <br>
            Ile (I) 26 6.1% <br>
            Leu (L) 39 9.2% <br>
            Lys (K) 27 6.4% <br>
            Met (M) 1 0.2% <br>
            Phe (F) 20 4.7% <br>
            Pro (P) 5 1.2% <br>
            Ser (S) 61 14.4% <br>
            Thr (T) 40 9.5% <br>
            Trp (W) 0 0.0% <br>
            Tyr (Y) 2 0.5% <br>
            Val (V) 29 6.9% <br>
            <br>
            negatively charged residues (Asp + Glu): 38 <br>
            positively charged residues (Arg + Lys): 30 <br>
            <br>
            Instability index: <br>
            The instability index (II) is computed to be 31.00 <br>
            This classifies the protein as <strong>stable. </strong></p>
              <p class="Estilo8">Aliphatic index: 89.50 </p>
              <p class="Estilo8">&nbsp;</p></td>
          <td width="171" valign="top"><p class="Estilo8">Number of amino acids: 185 <br>
            Molecular weight: 19266.4 <br>
            Theoretical pI: 4.29 <br>
            Amino acid composition: <br>
            Ala (A) 18 9.7% <br>
            Arg (R) 0 0.0% <br>
            Asn (N) 17 9.2% <br>
            Asp (D) 10 5.4% <br>
            c
            ys (C) 2 1.1% <br>
            Gln (Q) 9 4.9% <br>
            Glu (E) 8 4.3% <br>
            Gly (G) 12 6.5% <br>
            His (H) 1 0.5% <br>
            Ile (I) 12 6.5% <br>
            Leu (L) 18 9.7% <br>
              Lys (K) 8 4.3% <br>
              Met (M) 0 0.0% <br>
              Phe (F) 7 3.8% <br>
              Pro (P) 2 1.1% <br>
              Ser (S) 17 9.2% <br>
              Thr (T) 25 13.5% <br>
              Trp (W) 0 0.0% <br>
              Tyr (Y) 2 1.1% <br>
              Val (V) 17 9.2% </p>
            <p class="Estilo8">                (Asp + Glu): 18 <br>
                (Arg + Lys): 8 </p>            
            <p class="Estilo8">Instability index: <br>
              The instability index (II) is computed to be 18.52 <br>
              This classifies the protein as stable. </p>
              <p class="Estilo8">Aliphatic index: 99.62 </p>              </td>
          <td width="155" valign="top"><p class="Estilo8">Number of amino acids: 166 <br>
            Molecular weight: 17607.7 <br>
            Theoretical pI: 7.83 <br>
            Amino acid composition: <br>
            Ala (A) 7 4.2% <br>
            Arg (R) 3 1.8% <br>
            Asn (N) 14 8.4% <br>
            Asp (D) 9 5.4% <br>
            Cys (C) 3 1.8% <br>
            Gln (Q) 6 3.6% <br>
            Glu (E) 8 4.8% <br>
            Gly (G) 15 9.0% <br>
            His (H) 0 0.0% <br>
            Ile (I) 11 6.6% <br>
            Leu (L) 17 10.2% <br>
            Lys (K) 15 9.0% <br>
            Met (M) 0 0.0% <br>
            Phe (F) 10 6.0% <br>
            Pro (P) 3 1.8% <br>
            Ser (S) 26 15.7% <br>
            Thr (T) 11 6.6% <br>
            Trp (W) 0 0.0% <br>
            Tyr (Y) 0 0.0% <br>
            Val (V) 8 4.8% <br>
            <br>
            (Asp + Glu): 17 <br>
            (Arg + Lys): 18 </p>
              <p class="Estilo8">Instability index: <br>
              The instability index (II) is computed to be 42.42 <br>
              This classifies the protein as unstable. </p>
              <p class="Estilo8">Aliphatic index: 83.98 </p>              </td>
        </tr>
      </table>
      <p align="left" class="Estilo29"><br>
      </p>      
      <p class="Estilo6"><strong>Representaci&oacute;n Gr&aacute;fica predicci&oacute;n de glicosilaci&oacute;n CaEcm33p </strong></p>
      <span class="Estilo9"><strong>&gt;CaEcm33.3|Ecm33p_Candida </strong><br>
      MQFKNILAITAIAGLASAA <strong>NNST </strong>LTTATPSVKSECSFKDFTATASNQVQQVAACATAVGD ITIQGESFGTVELTGVQQIYGSLHVK <strong>NATK </strong>ATTINAATLQLVSGVLEFNANTILSDLNLA QLTTVGSFNLNALPALEKTGLTAGITSAESVVISDTGLSSLTGINVFKLKVFNVNNNDNI DTIDSGLQEVTDTIDISYNAEKVDVILKQLSSAKNIVLQQVNSFSAP <strong>NLTI </strong>A <strong>NGSF </strong>AVSS CSVEKVELAELTSIGNSLTINKNDDLTELDFPKLKTIGGALQISDNSELRSFSGFPKLES IGGSVNI <strong>NGTF </strong>D <strong>NGTF </strong>ESLKRVSGGFILKTDGKLSCSAFNKLNTNGDIKGDKFQCEDRQT TSSSSSKKSGSATETSGSSETGSDSSSSASGSSSSSKKGGAASNNGKLASVVAAFAAVGV ALF
      </span>      <p align="left" class="Estilo29">&nbsp;      </p>      <p align="left" class="Estilo23">&nbsp;</p>
<p align="left" class="Estilo11"><strong>&gt;CaPst1</strong></p>
<p><span class="Estilo13"><strong></strong></span><span class="Estilo4"><span class="Estilo27">Contexto gen&eacute;tico <br>
</span></span><span class="Estilo8"><strong>Pst1 se encuentra en el Cromosoma IV de S. cerevisiae bajo el siguiente contexto: </strong><br>
Region Displayed: 548K-578K bp Download/View Sequence </span></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="89" valign="top"><p class="Estilo23">RPC11 </p></td>
    <td width="560" valign="top"><p class="Estilo23">RNA polymerase III subunit C11; mediates pol III RNA cleavage activity and is important for termination of transcription </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>BAP3 </strong></p></td>
    <td width="560" valign="top"><p class="Estilo23"><strong>Amino acid permease involved in the uptake of cysteine, leucine, isoleucine and valine </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">HEM12 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Uroporphyrinogen decarboxylase, catalyzes the fifth step in the heme biosynthetic pathway; localizes to both the cytoplasm and nucleus; activity inhibited by Cu2+, Zn2+, Fe2+, Fe3+ and sulfhydryl-specific reagents </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YDR049W </p></td>
    <td width="560" valign="top"><p class="Estilo23">Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>TPI1 </strong></p></td>
    <td width="560" valign="top"><p class="Estilo23"><strong>induced under stress conditions; triosephosphate isomerase </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YDR051C </p></td>
    <td width="560" valign="top"><p class="Estilo23">Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">DBF4 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Regulatory subunit of Cdc7p-Dbf4p kinase complex, required for Cdc7p kinase activity and initiation of DNA replication; phosphorylates the Mcm2-7 family of proteins; cell cycle regulated </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>CDC34 </strong></p></td>
    <td width="560" valign="top"><p class="Estilo23"><strong>Ubiquitin-conjugating enzyme or E2; together with Skp1p, Rbx1p, Cdc53p, and an F-box protein, forms a ubiquitin-protein ligase called the SCF complex which regulates cell cycle progression by targeting key substrates for degradation </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>PST1 </strong></p></td>
    <td width="560" valign="top"><p class="Estilo23"><strong>Cell wall protein that contains a putative GPI-attachment site; secreted by regenerating protoplasts; up-regulated by activation of the cell integrity pathway, as mediated by Rlm1p; upregulated by cell wall damage via disruption of FKS1 </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YDR056C </p></td>
    <td width="560" valign="top"><p class="Estilo23">Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YOS9 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Glycoprotein of the ER membrane, involved in ER-to-Golgi transport of glycosylphosphatidylinositol (GPI)-anchored proteins </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">TGL2 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Triglyceride Lipase </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">TR(UCU)D </p></td>
    <td width="560" valign="top"><span class="Estilo9"></span></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">TD(GUC)D </p></td>
    <td width="560" valign="top"><span class="Estilo9"></span></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>UBC5 </strong></p></td>
    <td width="560" valign="top"><p class="Estilo23"><strong>Ubiquitin-conjugating enzyme that mediates selective degradation of short-lived and abnormal proteins, central component of the cellular stress response; expression is heat inducible </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">MAK21 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Protein required for large (60S) ribosomal subunit biogenesis; involved in nuclear export of pre-ribosomes; required for maintenance of dsRNA virus; homolog of human CAATT-binding protein </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YDR061W </p></td>
    <td width="560" valign="top"><p class="Estilo23">Mitochondrial protein, member of the ATP-binding cassette (ABC) transporter family; transcriptionally activated by Yrm1p along with genes involved in multidrug resistance </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">LCB2 </p></td>
    <td width="560" valign="top"><p class="Estilo23">Component of serine palmitoyltransferase, responsible along with Lcb1p for the first committed step in sphingolipid synthesis, which is the condensation of serine with palmitoyl-CoA to form 3-ketosphinganine </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">YDR063W </p></td>
    <td width="560" valign="top"><p class="Estilo23">Hypothetical ORF </p></td>
  </tr>
</table><p>&nbsp;</p>
<p><span class="Estilo13"><strong>Composici</strong><strong>&oacute;n fisico-qu&iacute;</strong><strong>mica </strong></span><strong>Pst1 </strong><strong></strong></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="206" valign="top"><p align="center" class="Estilo23"><strong>1-444 </strong></p></td>
    <td width="193" valign="top"><p align="center" class="Estilo23"><strong>19-199 </strong></p></td>
    <td width="179" valign="top"><p align="center" class="Estilo23"><strong>200-418 </strong></p></td>
  </tr>
  <tr>
    <td width="206" valign="top"><p class="Estilo23">Number of amino acids: 444 <br>
      Molecular weight: 45776.5 <br>
      Theoretical pI: 9.25 <br>
      Amino acid composition: <br>
      Ala (A) 36 8.1% <br>
      Arg (R) 1 0.2% <br>
      Asn (N) 36 8.1% <br>
      Asp (D) 17 3.8% <br>
      Cys (C) 4 0.9% <br>
      Gln (Q) 14 3.2% <br>
      Glu (E) 8 1.8% <br>
      Gly (G) 30 6.8% <br>
      His (H) 3 0.7% <br>
      Ile (I) 33 7.4% <br>
      Leu (L) 47 10.6% <br>
      Lys (K) 33 7.4% <br>
      Met (M) 4 0.9% <br>
      Phe (F) 18 4.1% <br>
      Pro (P) 5 1.1% <br>
      Ser (S) 87 19.6% <br>
      Thr (T) 40 9.0% <br>
      Trp (W) 1 0.2% <br>
      Tyr (Y) 3 0.7% <br>
      Val (V) 24 5.4% <br>
      <br>
      Negatively charged residues (Asp + Glu): 25 <br>
      Positively charged residues (Arg + Lys): 34 <br>
      <br>
      Instability index: <br>
      The instability index (II) is computed to be 36.15 <br>
      This classifies the protein as stable. <br>
      <br>
      Aliphatic index: 94.05 </p>        </td>
    <td width="193" valign="top"><p class="Estilo23">Number of amino acids: 181 <br>
      Molecular weight: 18844.0 <br>
      Theoretical pI: 6.80 <br>
      Amino acid composition: <br>
      Ala (A) 14 7.7% <br>
      Arg (R) 1 0.6% <br>
      Asn (N) 16 8.8% <br>
      Asp (D) 9 5.0% <br>
      Cys (C) 2 1.1% <br>
      Gln (Q) 6 3.3% <br>
      Glu (E) 3 1.7% <br>
      Gly (G) 9 5.0% <br>
      His (H) 1 0.6% <br>
      Ile (I) 14 7.7% <br>
      Leu (L) 19 10.5% <br>
      Lys (K) 11 6.1% <br>
      Met (M) 0 0.0% <br>
      Phe (F) 8 4.4% <br>
      Pro (P) 3 1.7% <br>
      Ser (S) 35 19.3% <br>
      Thr (T) 19 10.5% <br>
      Trp (W) 0 0.0% <br>
      Tyr (Y) 2 1.1% <br>Val (V) 9 5.0% </p>
        <p class="Estilo23">(Asp + Glu): 12 <br>
          (Arg + Lys): 12 </p>
        <p class="Estilo23">Instability index: <br>
          The instability index (II) is computed to be 47.70 <br>This classifies the protein as <strong>unstable. </strong></p>
        <p class="Estilo23">Aliphatic index: 93.26 </p>
        <p class="Estilo23">Grand average of hydropathicity (GRAVY): 0.020 </p></td>
    <td width="179" valign="top"><p class="Estilo23">Number of amino acids: 219<br>
      Molecular weight: 22462.0 <br>
      Theoretical pI: 9.59 <br>
      Amino acid composition: <br>Ala (A) 15 6.8% <br>
      Arg (R) 0 0.0% <br>
      Asn (N) 19 8.7% <br>
      Asp (D) 8 3.7% <br>
      Cys (C) 2 0.9% <br>
      Gln (Q) 6 2.7% <br>
      Glu (E) 5 2.3% <br>
      Gly (G) 19 8.7% <br>
      His (H) 1 0.5% <br>
      Ile (I) 12 5.5% <br>
      Leu (L) 20 9.1% <br>
      Lys (K) 22 10.0% <br>
      Met (M) 0 0.0% <br>
      Phe (F) 10 4.6% <br>
      Pro (P) 0 0.0% <br>
      Ser (S) 48 21.9% <br>
      Thr (T) 16 7.3% <br>
      Trp (W) 1 0.5% <br>
      Tyr (Y) 1 0.5% <br>
      Val (V) 14 6.4% <br>
      <br>
      (Asp + Glu): 13 <br>
      (Arg + Lys): 22 </p>
      <p class="Estilo23">Instability index: <br>
        The instability index (II) is computed to be 29.88<br>
        This classifies the protein as <strong>stable. </strong></p>
        <p class="Estilo23">Aliphatic index: 82.37 </p>
        <p class="Estilo23">Grand average of hydropathicity (GRAVY): -0.149 </p></td>
  </tr>
</table><p>&nbsp;</p>
<p><strong><span class="Estilo6"><strong>Representaci&oacute;n Gr&aacute;fica predicci&oacute;n de glicosilaci&oacute;n</strong></span><br>
  &gt;SaPst1|PST1_YEAST </strong></p>
<p class="Estilo9">MQLHSLIASTALLITSALAATSSSSSIPSSCTISSHATATAQSDLDKYSRCDTLVG <strong>NLTI <br>
</strong>GGGLKTGALANVKEI <strong>NGSL </strong>TIF <strong>NATN </strong>LTSFAADSLESITDSLNLQSLTILTSASFGSLQS <br>
VDSIKLITLPAISSFTSNIKSANNIYISDTSLQSVDGFSALKKVNVFNVNNNKKLTSIKS <br>
PVETVSDSLQFSFNG <strong>NQTK </strong>ITFDDLVWAN <strong>NISL </strong>TDVHSVSFANLQKI <strong>NSSL </strong>GFI <strong>NNSI </strong>SS <br>
L <strong>NFTK </strong>LNTIGQTFSIVSNDYLK <strong>NLSF </strong>S <strong>NLST </strong>IGGALVVA <strong>NNTG </strong>LQKIGGLD <strong>NLTT </strong>IGGTL <br>
EVVG <strong>NFTS </strong>LNLDSLKSVKGGADVESKSS <strong>NFSC </strong>NALKALQKKGGIKGESFVCKNGASSTSV <br>
KLSSTSKSQSSQTTAKVSKSSSKAEEKKFTSGDIKAAASASSVSSSGASSSSSKSSKGNA <br>
<span class="Estilo9">AIMAPIGQTTPLVGLLTAIIMSIM
</span>
</p>
<p align="left" class="Estilo23">&nbsp;</p>
<p align="left" class="Estilo23">&nbsp;</p>
<p><span class="Estilo4"><strong>Interacciones<br>
</strong></span><span class="Estilo9"><strong>BIND </strong><br>
  Swi5 Mlp1 Nic96 Nup116 Nup2 Prp20 Xpo1 </span></p>
<p class="Estilo4">&nbsp;</p><p class="Estilo4"><strong>&gt; <span class="Estilo31"><a name="CaPst1">CaPst1 </a>|Ecm33.1 </span></strong></p>
<p class="Estilo4"><strong>Contexto gen&eacute;tico<br>
</strong><span class="Estilo23">Genes in contig Chr5_Ctg10202 in the region of 'ECM331' (from 50.0 kb to 100.0 kb) (50000 bp) (22 matchs) </span></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6886.3f </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function, 3-prime end </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6886.5f </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function, 5-prime end </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF20009 </p></td>
    <td width="518" valign="top"><p class="Estilo23">Unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6881 </p></td>
    <td width="518" valign="top"><p class="Estilo23">putative phosphatidyl synthase </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6880 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6878 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6876 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6874.3f </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function, 3-prime end </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6872 </p></td>
    <td width="518" valign="top"><p class="Estilo23">serine/threonine protein kinase (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF6871 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23"><strong>ECM331 </strong></p></td>
    <td width="518" valign="top"><p class="Estilo23"><strong>Involved in cell wall biogenesis and architecture (by homology) </strong></p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">INT1 </p></td>
    <td width="518" valign="top"><p class="Estilo23">integrin-like protein alpha chain </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">FIP1 </p></td>
    <td width="518" valign="top"><p class="Estilo23">component of a pre-mRNA polyadenylation factor that interacts with poly(A) polymerase (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">TIF5 </p></td>
    <td width="518" valign="top"><p class="Estilo23">Translation initiation factor eIF5 (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">PUF2.3f </p></td>
    <td width="518" valign="top"><p class="Estilo23">RNA-binding protein, 3-prime end (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">PUF2.5f </p></td>
    <td width="518" valign="top"><p class="Estilo23">RNA-binding protein 5 prime end (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF9520 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">UAP1 </p></td>
    <td width="518" valign="top"><p class="Estilo23">UDP-N-acetylglucosamine pyrophosphorylase </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF9522.3f </p></td>
    <td width="518" valign="top"><p class="Estilo23">septin, 3-prime end (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF9522.53f </p></td>
    <td width="518" valign="top"><p class="Estilo23">septin, internal fragment (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF9522.5f </p></td>
    <td width="518" valign="top"><p class="Estilo23">septin, 5-prime end (by homology) </p></td>
  </tr>
  <tr>
    <td width="89" valign="top"><p class="Estilo23">IPF9525 </p></td>
    <td width="518" valign="top"><p class="Estilo23">unknown function </p></td>
  </tr>
</table>
<p>&nbsp;</p>
<p><span class="Estilo13"><strong>Composici</strong><strong>&oacute;n fisico-qu&iacute;</strong><strong>mica </strong></span><strong> CaPst1 </strong><strong></strong></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="216" valign="top"><p align="center" class="Estilo8"><strong>1-413 </strong></p></td>
    <td width="252" valign="top"><p align="center"><strong><span class="Estilo4"><span class="Estilo5"></span></span></strong></p></td>
    <td width="204" valign="top"><p align="center"><strong><span class="Estilo4"><span class="Estilo5"></span></span></strong></p></td>
  </tr>
  <tr>
    <td width="216" valign="top"><p class="Estilo8">Number of amino acids: 413<br>
      Molecular weight: 43628.0 <br>
      Theoretical pI: 4.69 <br>
      Amino acid composition: <br>
      Ala (A) 23 5.6% <br>
      Arg (R) 0 0.0% <br>
      Asn (N) 38 9.2% <br>
      Asp (D) 23 5.6% <br>
      Cys (C) 4 1.0% <br>
      Gln (Q) 16 3.9% <br>
      Glu (E) 16 3.9% <br>
      Gly (G) 25 6.1% <br>
      His (H) 4 1.0% <br>
      Ile (I) 30 7.3% <br>
      Leu (L) 52 12.6% <br>
      Lys (K) 25 6.1% <br>
      Met (M) 5 1.2% <br>
      Phe (F) 17 4.1% <br>
      Pro (P) 9 2.2% <br>
      Ser (S) 66 16.0% <br>
      Thr (T) 32 7.7% <br>
      Trp (W) 1 0.2% <br>
      Tyr (Y) 2 0.5% <br>
      Val (V) 25 6.1% <br>
      <br>
      (Asp + Glu): 39<br>
      (Arg + Lys): 25 </p>
        <p class="Estilo8">Instability index: <br>
          The instability index (II) is computed to be 28.32 <br>
          This classifies the protein as <strong>stable. <br>
          </strong></p>
        <p class="Estilo8">Aliphatic index: 100.56 </p>        <p class="Estilo8">Grand average of hydropathicity (GRAVY): 0.017 </p></td>
    <td width="252" valign="top"><p class="Estilo8">Number of amino acids: 178 <br>
      Molecular weight: 18707.9 <br>
      Theoretical pI: 4.33 <br>
      Amino acid composition: <br>
      Ala (A) 11 6.2% <br>
      Arg (R) 0 0.0% <br>
      Asn (N) 22 12.4% <br>
      Asp (D) 10 5.6% <br>
      Cys (C) 2 1.1% <br>
      Gln (Q) 9 5.1% <br>
      Glu (E) 5 2.8% <br>
      Gly (G) 9 5.1% <br>
      His (H) 0 0.0% <br>
      Ile (I) 18 10.1% <br>
      Leu (L) 24 13.5% <br>
      Lys (K) 8 4.5% <br>
      Met (M) 0 0.0% <br>
      Phe (F) 6 3.4% <br>
      Pro (P) 2 1.1% <br>
      Ser (S) 26 14.6% <br>
      Thr (T) 16 9.0% <br>
      Trp (W) 0 0.0% <br>
      Tyr (Y) 0 0.0% <br>
      Val (V) 10 5.6% <br>
      <br>
      (Asp + Glu): 15 <br>
          (Arg + Lys): 8 </p>
      <p class="Estilo8">Instability index: </p>
        <p class="Estilo8">The instability index (II) is computed to be 34.40 <br>
          This classifies the protein as <strong>stable. <br>
          </strong>Aliphatic index: 114.49 <br>
          Grand average of hydropathicity (GRAVY): 0.139 </p>        </td>
    <td width="204" valign="top"><p class="Estilo8">Number of amino acids: 192 <br>
      Molecular weight: 20606.6 <br>
      Theoretical pI: 4.93 <br>
      Amino acid composition: <br>
      Ala (A) 6 3.1% <br>
      Arg (R) 0 0.0% <br>
      Asn (N) 15 7.8% <br>
      Asp (D) 13 6.8% <br>
      Cys (C) 2 1.0% <br>
      Gln (Q) 6 3.1% <br>
      Glu (E) 11 5.7% <br>
      Gly (G) 14 7.3% <br>
      His (H) 4 2.1% <br>
      Ile (I) 9 4.7% <br>
      Leu (L) 18 9.4% <br>
      Lys (K) 16 8.3% <br>
      Met (M) 3 1.6% <br>
      Phe (F) 10 5.2% <br>
      Pro (P) 5 2.6% <br>
      Ser (S) 36 18.8% <br>
      Thr (T) 13 6.8% <br>
      Trp (W) 1 0.5% <br>
      Tyr (Y) 2 1.0% <br>
      Val (V) 8 4.2% </p>
      <p class="Estilo8">(Asp + Glu): 24 <br>
          (Arg + Lys): 16 </p>      
      <p class="Estilo8">Instability index: <br>
        The instability index (II) is computed to be 26.45 <br>
        This classifies the protein as <strong>stable. </strong></p>
        <p class="Estilo8">Aliphatic index: 70.05 </p>
        <p class="Estilo8">Grand average of hydropathicity (GRAVY): -0.498 </p></td>
  </tr>
</table>
<p>&nbsp;</p>
<p><span class="Estilo6"><strong>Representaci&oacute;n Gr&aacute;fica predicci&oacute;n de glicosilaci&oacute;n</strong></span></p>
<p> &gt;CaPst1|Ecm33.1 <span class="Estilo9">MQIKSFLLPIVAALLTSVSAADSSNKCSFSKTSITEATGISQLNACSTLDGEITVSGSGI GSIDLSSVKVLKAKLSILNSPSIVSLNFNQLQ <strong>NITG </strong>ALVIN <strong>NATQ </strong>LNSIDLTQLTNVETL QLVSLPSFAILNLNQGVQKAGTIVLSDTALTNLNGLASFNTIDSININNNK <strong>NISK </strong>IEFSD LQTVTDSLILSFNNDDAEVKLDSLKWAG <strong>NLTI </strong>QDVSSIQAS <strong>NLTS </strong>V <strong>NGSL </strong>LISYNTFDEL EFPNLKSVGNSMQIFAHDELTKISFPKLSELDGELEMF <strong>NNTQ </strong>LEEIDFG <strong>NLTT </strong>IKGAVTI<br> 
  SGPFD <strong>NLTM </strong>ENLKLVSGDFQV <strong>NSTS </strong>DKFDCSAFDKLHEKGKIEGHNYVCTHPANPSSSSK SGSSTQTGKSDSKSSDGSSSS <strong>NSSS </strong>SSKKGASNVLVVPGMVLTTALGVLLALI </span></p>
<p align="left" class="Estilo23">&nbsp;</p>
<p class="Estilo11"><strong>&gt; <a name="SaSps2">SaSps2 </a>|SPS2_YEAST </strong><strong></strong></p>
<p><span class="Estilo6">Contexto gen&eacute;tico </span></p>
<p><strong>Sps2 se encuentra tambi </strong><strong>&eacute; </strong><strong>n en el cromosoma IV bajo el siguiente contexto gen </strong><strong>&eacute; </strong><strong>tico: </strong></p>
<p>Region Displayed: 1,468K-1,499K bp Download/View Sequence </p>
<table cellspacing="0" cellpadding="0">
  <tr class="Estilo23">
    <td width="77" valign="top"><p><strong><em>SMT3 </em></strong></p></td>
    <td width="573" valign="top"><p><strong>Protein that may be involved in function and/or structure of the eukaryotic kinetochore; has similarity to SUMO-1; ubiquitin-like protein </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>ACN9 </em></p></td>
    <td width="573" valign="top"><p>Protein of the mitochondrial intermembrane space, required for acetate utilization and gluconeogenesis; has orthologs in higher eukaryotes </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>EMI1 </em></p></td>
    <td width="573" valign="top"><p><strong>Non-essential protein of unknown function required for transcriptional induction of the early meiotic-specific transcription factor IME1, also required for sporulation </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>TTR1 </em></p></td>
    <td width="573" valign="top"><p>Glutaredoxin (thioltransferase) (glutathione reductase) </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>YDR514C </em></p></td>
    <td width="573" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>SLF1 </em></p></td>
    <td width="573" valign="top"><p>Associates with translating ribosomes; may function in cytoplasm to modulate mRNA translation; regulates the copper-dependent mineralization of copper sulfide complexes on cell surface in cells cultured in medium containing copper salts; RNA binding </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>EMI2 </em></p></td>
    <td width="573" valign="top"><p><strong>Non-essential protein of unknown function required for transcriptional induction of the early meiotic-specific transcription factor IME1, also required for sporulation </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>GRH1 </em></p></td>
    <td width="573" valign="top"><p>Golgi localized protein component of the spindle assembly checkpoint; homolog of human GRASP65, which functions in postmitotic reassembly of Golgi stacks </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>EUG1 </em></p></td>
    <td width="573" valign="top"><p>Protein disulfide isomerase of the endoplasmic reticulum lumen, function overlaps with that of Pdi1p; may interact with nascent polypeptides in the ER </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>FPR2 </em></p></td>
    <td width="573" valign="top"><p><strong>Membrane-bound peptidyl-prolyl cis-trans isomerase (PPIase), binds to the drugs FK506 and rapamycin; expression pattern suggests possible involvement in ER protein trafficking </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>YDR520C </em></p></td>
    <td width="573" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><strong><em>SPS2 </em></strong></p></td>
    <td width="573" valign="top"><p><strong>Middle/late gene of meiosis; involved in meiosis </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><strong><em>SPS1 </em></strong></p></td>
    <td width="573" valign="top"><p><strong>dispensable for mitosis, involved in middle/late stage of meiosis, required for spore wall formation; serine/threonine kinase homologous to Ste20p; expressed in middle/late meiosis </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>AGE1 </em></p></td>
    <td width="573" valign="top"><p>ADP-ribosylation factor (ARF) GTPase activating protein (GAP) effector; ARF GAP with effector function(s) </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>YDR524W-A </em></p></td>
    <td width="573" valign="top"><p>Identified by SAGE </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>YDR524C-B </em></p></td>
    <td width="573" valign="top"><p>Ydr524c-bp </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>SNA2 </em></p></td>
    <td width="573" valign="top"><p>Protein of unknown function, has similarity to Pmp3p, which is involved in cation transport; green fluorescent protein (GFP)-fusion protein localizes to the cytoplasm in a punctate pattern </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>RBA50 </em></p></td>
    <td width="573" valign="top"><p><strong>Protein required for cell viability </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><strong><em>HLR1 </em></strong></p></td>
    <td width="573" valign="top"><p><strong>Protein involved in regulation of cell wall composition and integrity and response to osmotic stress; overproduction suppresses a lysis sensitive PKC mutation; similar to Lre1p, which functions antagonistically to protein kinase A </strong></p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>QCR7 </em></p></td>
    <td width="573" valign="top"><p>ubiquinol-cytochrome c oxidoreductase subunit 7 (14 kDa) </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>APA2 </em></p></td>
    <td width="573" valign="top"><p>Diadenosine 5'',5''''-P1,P4-tetraphosphate phosphorylase II (AP4A phosphorylase), involved in catabolism of bis(5''-nucleosidyl) tetraphosphates; has similarity to Apa1p </p></td>
  </tr>
  <tr class="Estilo23">
    <td width="77" valign="top"><p><em>YDR531W </em></p></td>
    <td width="573" valign="top"><p>Pantothenate kinase (ATP:D-pantothenate 4''-phosphotransferase, EC 2.7.1.33) catalyzes the first committed step in the universal biosynthetic pathway leading to CoA </p></td>
  </tr>
</table><p>&nbsp;</p>
<p class="Estilo6"><strong>Composici&oacute;n fisico-qu&iacute;mica SaSps2 </strong><strong></strong></p>
<table cellspacing="0" cellpadding="0">
  <tr class="Estilo8">
    <td width="187" valign="top"><p align="center" class="Estilo36 Estilo37">1-502 </p></td>
    <td width="180" valign="top"><p align="center" class="Estilo38">57-274 </p></td>
    <td width="164" valign="top"><p align="center" class="Estilo38">275-477 </p></td>
  </tr>
  <tr class="Estilo8">
    <td width="187" valign="top"><p class="Estilo37">Number of amino acids: 502 <br>
      Molecular weight: 55939.4 <br>
      Theoretical pI: 6.59 <br>
      Amino acid composition: <br>
      Ala (A) 21 4.2% <br>
      Arg (R) 8 1.6% <br>
      Asn (N) 44 8.8% <br>
      Asp (D) 24 4.8% <br>
      Cys (C) 4 0.8% <br>
      Gln (Q) 16 3.2% <br>
      lu (E) 37 7.4% <br>
      Gly (G) 29 5.8% <br>
      His (H) 5 1.0% <br>
      le (I) 49 9.8% <br>
      Leu (L) 55 11.0% <br>
      Lys (K) 52 10.4% <br>
      Met (M) 9 1.8% <br>
      Phe (F) 26 5.2% <br>
      pro (P) 12 2.4% <br>
      Ser (S) 46 9.2% <br>
      Thr (T) 26 5.2% <br>
      Trp (W) 3 0.6% <br>
      Tyr (Y) 7 1.4% <br>
      Val (V) 29 5.8% </p>
      <p class="Estilo37">(Asp + Glu): 61 <br>
        (Arg + Lys): 60 </p>
        <p class="Estilo37">Instability index: <br>
          The instability index (II) is computed to be 41.06 <br>
          This classifies the protein as <strong>unstable. </strong></p>
        <p class="Estilo37">Aliphatic index: 101.73 </p>
        <p class="Estilo37">Grand average of hydropathicity (GRAVY): -0.173 </p></td>
    <td width="180" valign="top"><p class="Estilo37">Number of amino acids: 218 <br>
      Molecular weight: 24422.9 <br>
      Theoretical pI: 5.21 <br>
      Amino acid composition: <br>
      Ala (A) 8 3.7% <br>
      Arg (R) 4 1.8% <br>
      Asn (N) 23 10.6% <br>
      Asp (D) 12 5.5% <br>
      Cys (C) 2 0.9% <br>
      Gln (Q) 6 2.8% <br>
      Glu (E) 17 7.8% <br>
      Gly (G) 11 5.0% <br>
      His (H) 4 1.8% <br>
      Ile (I) 25 11.5% <br>
      Leu (L) 24 11.0% <br>
      Lys (K) 18 8.3% <br>
      Met (M) 1 0.5% <br>
      Phe (F) 8 3.7% <br>
      Pro (P) 6 2.8% <br>
      Ser (S) 16 7.3% <br>
      Thr (T) 11 5.0% <br>
      Trp (W) 2 0.9% <br>
      Tyr (Y) 4 1.8% <br>
      Val (V) 16 7.3% </p>
      <p class="Estilo37">(Asp + Glu): 29 <br>
          (Arg + Lys): 22 </p>      
      <p class="Estilo37">Instability index: <br>
        The instability index (II) is computed to be 44.37 <br>
        This classifies the protein as <strong>unstable. </strong></p>
        <p class="Estilo37">Aliphatic index: 112.61 </p>
        <p class="Estilo37">Grand average of hydropathicity (GRAVY): -0.142 </p></td>
    <td width="164" valign="top"><p class="Estilo37">Number of amino acids: 203 <br>
      Molecular weight: 22109.1 <br>
      Theoretical pI: 6.84 <br>
      Amino acid composition: <br>
      Ala (A) 10 4.9% <br>
      Arg (R) 3 1.5% <br>
      Asn (N) 17 8.4% <br>
      Asp (D) 10 4.9% <br>
      Cys (C) 2 1.0% <br>
      Gln (Q) 6 3.0% <br>
      Glu (E) 18 8.9% <br>Gly (G) 18 8.9% <br>
      His (H) 1 0.5% <br>
      Ile (I) 14 6.9% <br>
      Leu (L) 23 11.3% <br>
      Lys (K) 25 12.3% <br>
      Met (M) 5 2.5% <br>
      Phe (F) 8 3.9% <br>
      Pro (P) 4 2.0% <br>
      Ser (S) 22 10.8% <br>
      Thr (T) 8 3.9% <br>
      Trp (W) 0 0.0% <br>
      Tyr (Y) 2 1.0% <br>
      Val (V) 7 3.4% </p>
        <p class="Estilo37">(Asp + Glu): 28 <br>
          (Arg + Lys): 28 </p>
        <p class="Estilo37"> Instability index: <br>
          The instability index (II) is computed to be 35.99 <br>
          This classifies the protein as <strong>stable. <br>
          </strong>Aliphatic index: 86.01 </p>        
        <p class="Estilo37">Grand average of hydropathicity (GRAVY): -0.480 </p></td>
  </tr>
</table><p>&nbsp;</p>
<p class="Estilo6"><strong>Representaci&oacute;n Gr&aacute;fica predicci&oacute;n de glicosilaci&oacute;n SaSps2</strong></p>
<span class="Estilo9">&gt;SaSps2|SPS2_YEAST MPIWKTQTFFTSISVIQIVNKETKVSTKKEKDSMLNQLNTILRFLFLFLQLIKSSAAVEP NGGPNILDHNIMLVNT <strong>NATI </strong>PKKEQTDFEVISPTKQTQVDEDCKKGLYHIENAGNLIELQ AKCWKVVGNIEISS <strong>NYSG </strong>SLIDLGLIREIEGDLIIKNNKHIFRIQGYNLESLGKLELDSL TSFVSLDFPALKEVETVDWRVLPILSSVVINGNIKKIKNIIISDTALTSIDYFNNVKKVD IFNINNNRFLENLFASLESVTKQLTVHSNAKELELDLSNLHTVE <strong>NMTI </strong>KDVSEIKLAKLS SV <strong>NSSL </strong>EFIENQFSSLELPLLAKVQGTLGLIDNKNLKKL <strong>NFSN </strong>ATDIQGGLMIA <strong>NNTE </strong>LA KIDFFPKLRQIGGAIYFEGSFDKIDLPELKLVKGSAYIKSSSEELNCEEFTSPKAGRSII RGGKIECTSGMKSKMLNVDEEGNVLGKQETDNDNGKKEKGKNGAKSQGSSKKMENSAPKN IFIDAFKMSVYAVFTVLFSIIF
</span><p align="left" class="Estilo23">&nbsp;</p>
<p class="Estilo6"><strong>Interacciones </strong><strong></strong></p>
<p><strong>BIND </strong><strong></strong></p>
<p><a href="http://bind.ca/Action?textquery=%22PI%5B3%5DP%20ammonium%22">PI[3]P ammonium </a><a href="http://bind.ca/Action?idsearch=gi:6323539">Arg81 </a><a href="http://bind.ca/Action?idsearch=gi:6324934">Hap5 </a><a href="http://bind.ca/Action?idsearch=gi:6324606">Hms1 </a><a href="http://bind.ca/Action?pg=3001&identifier=bindid&idsearch=21537"></a><a href="http://bind.ca/Action?idsearch=gi:6323484">Leu3 </a><a href="http://bind.ca/Action?idsearch=gi:6325219">Met31 </a><a href="http://bind.ca/Action?idsearch=gi:6319365">Rtg3 </a><a href="http://bind.ca/Action?pg=3001&identifier=bindid&idsearch=21540"></a><a href="http://bind.ca/Action?idsearch=gi:6320516">Sum1 </a><a href="http://bind.ca/Action?pg=3001&identifier=bindid&idsearch=131455"></a><a href="http://bind.ca/Action?idsearch=gi:6320996">TBP1 </a><a href="http://bind.ca/Action?pg=3001&identifier=bindid&idsearch=141023"></a><a href="http://bind.ca/Action?idsearch=gi:6319318">Nup60 </a><strong></strong></p>
<p>&nbsp; </p>
<p><span class="Estilo11"><strong>&gt; <a name="SaSps22">SaSps22</a></strong></span> </p><p>&nbsp;</p>
<p><strong>Contexto gen &eacute; tico </strong></p><p>localizada en el cromosoma III de Saccharomyces cerevisiae, rodeada por los siguientes genes: </p>
<p><strong>Region Displayed: 16K-67K bp Download/View Sequence </strong></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="74" valign="top"><p><em>CHA1 </em></p></td>
    <td width="494" valign="top"><p>catabolism of hydroxy amino acids; catabolic serine (threonine) dehydratase </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>VAC17 </em></p></td>
    <td width="494" valign="top"><p>vacuole inheritance </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>MRC1 </em></p></td>
    <td width="494" valign="top"><p><strong>S-phase checkpoint protein found at replication forks, required for DNA replication; also required for Rad53p activation during DNA replication stress, where it forms a replication-pausing complex with Tof1p and is phosphorylated by Mec1p; protein in </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><strong><em>KRR1 </em></strong></p></td>
    <td width="494" valign="top"><p><strong>Essential nucleolar protein required for the synthesis of 18S rRNA and for the assembly of 40S ribosomal subunit </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>FYV5 </em></p></td>
    <td width="494" valign="top"><p>Protein of unknown function, required for survival upon exposure to K1 killer toxin; involved in ion homeostasis </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>PRD1 </em></p></td>
    <td width="494" valign="top"><p>Zinc metalloendopeptidase, found in the cytoplasm and intermembrane space of mitochondria </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL056C </em></p></td>
    <td width="494" valign="top"><p>Protein of unknown function; green fluorescent protein (GFP)-fusion protein localizes to the cytoplasm in a punctate pattern </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>KAR4 </em></p></td>
    <td width="494" valign="top"><p><strong>Transcription factor required for induction of KAR3 and CIK1 during mating, also required during meiosis; exists in two forms, a slower-migrating form more abundant during vegetative growth and a faster-migrating form induced by pheromone </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>SPB1 </em></p></td>
    <td width="494" valign="top"><p>Suppressor of PaB1 mutant; encodes a S-adenosylmethionine-dependent methyltransferase; involved in 60S ribosomal subunit biogenesis and rRNA processing; Putative methyltransferase </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>PBN1 </em></p></td>
    <td width="494" valign="top"><p>Protein required for the autocatalytic post-translational processing of the protease B precursor Prb1p; localizes to the ER in a lumenal orientation </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><strong><em>LRE1 </em></strong></p></td>
    <td width="494" valign="top"><p><strong>Protein involved in control of cell wall structure and stress response; inhibits Cbk1p protein kinase activity; overproduction confers resistance to cell-wall degrading enzymes </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>APA1 </em></p></td>
    <td width="494" valign="top"><p>Diadenosine 5'',5''''-P1,P4-tetraphosphate phosphorylase I (AP4A phosphorylase), involved in catabolism of bis(5''-nucleosidyl) tetraphosphates; has similarity to Apa2p </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL049C </em></p></td>
    <td width="494" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><strong><em>SPS22 </em></strong></p></td>
    <td width="494" valign="top"><p><strong>Required redundantly with SPS2 for organization of the beta-glucan layer of the spore wall </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL047C </em></p></td>
    <td width="494" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL045C </em></p></td>
    <td width="494" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL044C </em></p></td>
    <td width="494" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>PDI1 </em></p></td>
    <td width="494" valign="top"><p>Protein disulfide isomerase, multifunctional protein resident in the endoplasmic reticulum lumen, essential for the formation of disulfide bonds in secretory and cell-surface proteins, unscrambles non-native disulfide bonds </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>GLK1 </em></p></td>
    <td width="494" valign="top"><p>Glucokinase, catalyzes the phosphorylation of glucose at C6 in the first irreversible step of glucose metabolism; one of three glucose phosphorylating enzymes; expression regulated by non-fermentable carbon sources </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>GID7 </em></p></td>
    <td width="494" valign="top"><p>Protein of unknown function, involved in proteasome-dependent catabolite inactivation of fructose-1,6-bisphosphatase; contains six WD40 repeats; computational analysis suggests that Gid7p and Moh1p have similar functions </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>ATG22 </em></p></td>
    <td width="494" valign="top"><p>Autophagy gene essential for breakdown of autophagic vesicles in the vacuole </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><strong><em>SRO9 </em></strong></p></td>
    <td width="494" valign="top"><p><strong>Associates with translating ribosomes; may function in the cytoplasm to modulate mRNA translation; may be involved in organization of actin filaments; RNA binding protein with La motif </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>GFD2 </em></p></td>
    <td width="494" valign="top"><p>Protein of unknown function, identified as a high-copy suppressor of a dbp5 mutation </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><strong><em>GRX1 </em></strong></p></td>
    <td width="494" valign="top"><p><strong>Hydroperoxide and superoxide-radical responsive heat-stable glutathione-dependent disulfide oxidoreductase with active site cysteine pair; protects cells from oxidative damage </strong></p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>LSB5 </em></p></td>
    <td width="494" valign="top"><p>Protein of unknown function; binds Las17p, which is a homolog of human Wiskott-Aldrich Syndrome protein involved in actin patch assembly and actin polymerization </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>YCL033C </em></p></td>
    <td width="494" valign="top"><p>Hypothetical ORF </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>STE50 </em></p></td>
    <td width="494" valign="top"><p>Protein involved in mating response, invasive/filamentous growth, and osmotolerance, acts as an adaptor that links G protein-associated Cdc42p-Ste20p complex to the effector Ste11p to modulate signal transduction </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>RRP7 </em></p></td>
    <td width="494" valign="top"><p>involved in rRNA processing </p></td>
  </tr>
  <tr>
    <td width="74" valign="top"><p><em>HIS4 </em></p></td>
    <td width="494" valign="top"><p>Multifunctional enzyme containing phosphoribosyl-ATP pyrophosphatase, phosphoribosyl-AMP cyclohydrolase, and histidinol dehydrogenase activities; catalyzes the second, third, ninth and tenth steps in histidine biosynthesis </p></td>
  </tr>
</table><p><strong>Composici </strong><strong>&oacute; </strong><strong>n fisico-qu </strong><strong>&iacute; </strong><strong>mica SaSps22 </strong><strong></strong></p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="228" valign="top"><p align="center"><strong></strong></p></td>
    <td width="240" valign="top"><p align="center"><strong></strong></p></td>
    <td width="204" valign="top"><p align="center"><strong></strong></p></td>
  </tr>
  <tr>
    <td width="228" valign="top"><p>Number of amino acids: 463 </p>
        <p>Molecular weight: 52007.7 </p>
        <p>Theoretical pI: 5.92 </p>
        <p>Amino acid composition: </p>
        <p>Ala (A) 17 3.7% </p>
        <p>Arg (R) 15 3.2% </p>
        <p>Asn (N) 44 9.5% </p>
        <p>Asp (D) 23 5.0% </p>
        <p>Cys (C) 7 1.5% </p>
        <p>Gln (Q) 18 3.9% </p>
        <p>Glu (E) 30 6.5% </p>
        <p>Gly (G) 26 5.6% </p>
        <p>His (H) 9 1.9% </p>
        <p>Ile (I) 47 10.2% </p>
        <p>Leu (L) 56 12.1% </p>
        <p>Lys (K) 32 6.9% </p>
        <p>Met (M) 7 1.5% </p>
        <p>Phe (F) 25 5.4% </p>
        <p>Pro (P) 15 3.2% </p>
        <p>Ser (S) 37 8.0% </p>
        <p>Thr (T) 22 4.8% </p>
        <p>Trp (W) 2 0.4% </p>
        <p>Tyr (Y) 7 1.5% </p>
        <p>Val (V) 24 5.2% </p>
        <p>Asx (B) 0 0.0% </p>
        <p>Glx (Z) 0 0.0% </p>
        <p>Xaa (X) 0 0.0% </p>
        <p>(Asp + Glu): 53 </p>
        <p>(Arg + Lys): 47 </p>
        <p>Estimated half-life: </p>
        <p>The estimated half-life is: 30 hours </p>
        <p>&gt;20 hours (yeast, in vivo). </p>
        <p>&gt;10 hours (Escherichia coli, in vivo). </p>
        <p>Instability index: </p>
        <p>The instability index (II) is computed to be 44.15 </p>
        <p>This classifies the protein as <strong>unstable. </strong></p>
        <p>Aliphatic index: 105.46 </p>
        <p>Grand average of hydropathicity (GRAVY): -0.124 </p></td>
    <td width="240" valign="top"><p>Number of amino acids: 221 </p>
        <p>Molecular weight: 24984.3 </p>
        <p>Theoretical pI: 5.38 </p>
        <p>Amino acid composition: </p>
        <p>Ala (A) 8 3.6% </p>
        <p>Arg (R) 8 3.6% </p>
        <p>Asn (N) 23 10.4% </p>
        <p>Asp (D) 12 5.4% </p>
        <p>Cys (C) 3 1.4% </p>
        <p>Gln (Q) 11 5.0% </p>
        <p>Glu (E) 16 7.2% </p>
        <p>Gly (G) 9 4.1% </p>
        <p>His (H) 7 3.2% </p>
        <p>Ile (I) 24 10.9% </p>
        <p>Leu (L) 25 11.3% </p>
        <p>Lys (K) 12 5.4% </p>
        <p>Met (M) 1 0.5% </p>
        <p>Phe (F) 11 5.0% </p>
        <p>Pro (P) 9 4.1% </p>
        <p>Ser (S) 18 8.1% </p>
        <p>Thr (T) 8 3.6% </p>
        <p>Trp (W) 1 0.5% </p>
        <p>Tyr (Y) 2 0.9% </p>
        <p>Val (V) 13 5.9% </p>
        <p>Asx (B) 0 0.0% </p>
        <p>Glx (Z) 0 0.0% </p>
        <p>Xaa (X) 0 0.0% </p>
        <p>(Asp + Glu): 28 </p>
        <p>(Arg + Lys): 20 </p>
        <p>Estimated half-life: </p>
        <p>The estimated half-life is: 30 hours </p>
        <p>&gt;20 hours (yeast, in vivo). </p>
        <p>&gt;10 hours (Escherichia coli, in vivo). </p>
        <p>Instability index: </p>
        <p>The instability index (II) is computed to be 56.59 </p>
        <p>This classifies the protein as unstable. </p>
        <p>Aliphatic index: 107.15 </p>
        <p>Grand average of hydropathicity (GRAVY): -0.233 </p></td>
    <td width="204" valign="top"><p>Number of amino acids: 159 </p>
        <p>Molecular weight: 17775.4 </p>
        <p>Theoretical pI: 5.93 </p>
        <p>Amino acid composition: </p>
        <p>Ala (A) 3 1.9% </p>
        <p>Arg (R) 4 2.5% </p>
        <p>Asn (N) 16 10.1% </p>
        <p>Asp (D) 7 4.4% </p>
        <p>Cys (C) 2 1.3% </p>
        <p>Gln (Q) 4 2.5% </p>
        <p>Glu (E) 11 6.9% </p>
        <p>Gly (G) 13 8.2% </p>
        <p>His (H) 1 0.6% </p>
        <p>Ile (I) 17 10.7% </p>
        <p>Leu (L) 20 12.6% </p>
        <p>Lys (K) 13 8.2% </p>
        <p>Met (M) 3 1.9% </p>
        <p>Phe (F) 9 5.7% </p>
        <p>Pro (P) 4 2.5% </p>
        <p>Ser (S) 13 8.2% </p>
        <p>Thr (T) 8 5.0% </p>
        <p>Trp (W) 1 0.6% </p>
        <p>Tyr (Y) 3 1.9% </p>
        <p>Val (V) 7 4.4% </p>
        <p>Asx (B) 0 0.0% </p>
        <p>Glx (Z) 0 0.0% </p>
        <p>Xaa (X) 0 0.0% </p>
        <p>(Asp + Glu): 18 </p>
        <p>(Arg + Lys): 17 </p>
        <p>Estimated half-life: </p>
        <p>The estimated half-life is: 5.5 hours </p>
        <p>3 min (yeast, in vivo). </p>
        <p>2 min (Escherichia coli, in vivo). </p>
        <p>Instability index: </p>
        <p>The instability index (II) is computed to be 31.29 </p>
        <p>This classifies the protein as <strong>stable. </strong></p>
        <p>Aliphatic index: 105.41 </p>
        <p>Grand average of hydropathicity (GRAVY): -0.089 </p></td>
  </tr>
</table><p>&nbsp;</p>
<p><strong>Predicci </strong><strong>&oacute; </strong><strong>n P </strong><strong>&eacute; </strong><strong>ptido Se </strong><strong>&ntilde; </strong><strong>al y anclaje GPI </strong><strong></strong></p>
<p><strong>SignalP 3.0 </strong><strong></strong></p>
<p># Most likely cleavage site between pos. 25 and 26: ALG-AA </p>
<p><strong>PSORT II </strong><strong></strong></p>
<p>&gt;&gt;&gt; Seems to have a cleavable signal peptide (1 to 25) </p>
<p>k = 9/23 </p>
<p>44.4 %: extracellular, including cell wall </p>
<p>22.2 %: endoplasmic reticulum </p>
<p>11.1 %: vacuolar </p>
<p>11.1 %: Golgi </p>
<p>&#149;&nbsp; %: mitochondrial </p>
<p><strong>GPI Prediction Server </strong><strong></strong></p>
<p>Potential GPI-modification site was found. </p>
<p>Sequence position of omega-site : 440 </p>
<p><strong>DGPI results </strong><strong></strong></p>
<p>Conclusion : </p>
<p>This protein is GPI-anchored (signal, hydrophobic &amp; hydrophilic tail present). </p>
<p>There is a potential cleavage site at 440 (w, w+1, w+2 in red) </p>
<p><br>
  <strong>Representaci </strong><strong>&oacute; </strong><strong>n Gr </strong><strong>&aacute; </strong><strong>fica predicci </strong><strong>&oacute; </strong><strong>n de glicosilaci </strong><strong>&oacute; </strong><strong>n </strong></p>
<p class="Estilo9">&gt; SaSps22 MNRITRKSCLFAIIFASLFVTHALGAAIDPPRRPHNVKPFHNGNLELQRRANEPFFEIDV KSLNTNSPISELCKKDLHVIESSHDLFHLQNQCEFILGSLKVTNYDSNILDLNSLRAIGG DLIIQDSPELIRIQAGNLNKIEGLFQLQGLTSLVSVEIPTLKFCQSLEWKVVPILNYVSM DSQNIEIIKDIVISDTSLANIENFNKVQEIDTFNINNNRFLETIHSNVKTIRGQFSVHAN AKELELEMPHLREVE <strong>NITI </strong>RDTSLVYLPQLTKVKSSLEFIENYFYELNLNNLQKIGGTLG IINNVNLIKVNLE <strong>NLTD </strong>IQGGLMIAD <strong>NESL </strong>EDITFLPNLKQIGGAIFFEGSFKDIMFDSL KLVKGSAFIKSSSNVLDCNKWTNPSNGRSIIRGGKFTCISGKKENTLNVKQDGTIIEKGY KDLTQEGEDSKKRVISKYANSANPSMQLDPLLFGTCLVAMLLF
</p>
<p class="Estilo9">&nbsp;</p>
<p><strong>Interacciones </strong></p>
<p><strong>BIND </strong></p>
<p><a href="http://bind.ca/Action?idsearch=gi:6320516">SUM1 </a><a href="http://bind.ca/Action?idsearch=gi:6319318">Nup60 </a><a href="http://bind.ca/Action?idsearch=gi:6323575">Orc1 </a><a href="http://bind.ca/Action?idsearch=gi:6320516">Sum1 </a></p>
<p>&nbsp;</p>
<p><strong>&gt; <a name="SpMeu10">SpMeu10 </a></strong><strong>|SpMeu10Ecm33 </strong><strong></strong></p><p><strong>Meu 10 es hom </strong><strong>&oacute; </strong><strong>loga de YIN3_SCHPO </strong></p><p><strong>Predicci </strong><strong>&oacute; </strong><strong>n P </strong><strong>&eacute; </strong><strong>ptido Se </strong><strong>&ntilde; </strong><strong>al y anclaje GPI </strong><strong></strong></p>
<p><strong>SignalP 3.0 </strong><strong></strong></p>
<p># Most likely cleavage site between pos. 29 and 30: VCG-AP </p>
<p>Max cleavage site probability: 0.295 between pos. 25 and 26 </p>
<p><strong>PSORT II </strong><strong></strong></p>
<p>&gt;&gt;&gt; Seems to have no N-terminal signal peptide between 29 and 30 </p>
<p>k = 9/23 </p>
<p>22.2 %: vesicles of secretory system </p>
<p>22.2 %: plasma membrane </p>
<p>11.1 %: cytoplasmic </p>
<p>11.1 %: Golgi </p>
<p>11.1 %: nuclear </p>
<p>11.1 %: vacuolar </p>
<p>&#149;&nbsp; %: mitochondrial </p>
<p><strong>GPI Prediction Server </strong><strong></strong></p>
<p>None potential GPI-modification site was found. </p>
<p>Among all positions checked, sequence position 395 had the best score </p>
<p><strong>DGPI results </strong><strong></strong></p>
<p>This protein is GPI-anchored at 389 </p>
<p>&nbsp;</p>
<p><strong>&gt; <a name="EgSps22">EgSps22 </a>| </strong><strong>AFR723Cp [Eremothecium gossypii] Hom </strong><strong>&oacute; </strong><strong>loga de las sps2 y 22 </strong><strong></strong></p>
<p><strong>Predicci </strong><strong>&oacute; </strong><strong>n P </strong><strong>&eacute; </strong><strong>ptido Se </strong><strong>&ntilde; </strong><strong>al y anclaje GPI </strong><strong></strong></p>
<p><strong>SignalP 3.0 </strong><strong></strong></p>
<p>Signal peptide between pos. 19 and 20 </p>
<p><strong>PSORT II </strong><strong></strong></p>
<p>&gt;&gt;&gt; Seems to have a cleavable signal peptide (1 to 19) </p>
<p>k = 9/23 </p>
<p>34.8 %: plasma membrane </p>
<p>34.8 %: extracellular, including cell wall </p>
<p>17.4 %: endoplasmic reticulum </p>
<p>&#149;&nbsp; %: Golgi </p>
<p><strong>GPI Prediction server </strong><strong></strong></p>
<p>None potential GPI-modification site was found. </p>
<p>Among all positions checked, sequence position 457 had the best score </p>
<p><strong>DGPI Results </strong><strong></strong></p>
<p>N-term Signal : </p>
<p>there is a N-term signal (1..18 in violet) </p>
<p>Conclusion : </p>
<p>This protein is GPI-anchored at 455 (w, w+1, w+2 in red) </p>
<p>&nbsp;</p>
<p><strong>&gt; <a name="CaSps22">CaSps22 </a></strong><strong> |CA0513|IPF13972: 478 aa </strong><strong></strong></p>
<p><strong>Contexto gen </strong><strong>&eacute; </strong><strong>tico </strong></p>
<p>Un posible hom &oacute; logo de Sps22 de Saccharomyces cerevisiae es &gt;C. albicans fungus|CA0513|IPF13972: 478 aa - unknown function. De hecho, la homolog &iacute; a entre los dos es  </p>
<p>C. albicans fungus|CA0513|IPF13972 unknown function 225 1e-59 </p>
<p>Genes in contig Chr1_Ctg10216 in the region of 'IPF13972' (from 90.0 kb to 130.0 kb) (40000 bp) </p>
<table cellspacing="0" cellpadding="0">
  <tr>
    <td width="76" valign="top"><p>IPF7880 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p><strong>HIP1 </strong></p></td>
    <td width="574" valign="top"><p><strong>Histidine permease (by homology) </strong></p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>TYE7 </p></td>
    <td width="574" valign="top"><p>Basic helix-loop-helix transcription factor by homology </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>PSA2 </p></td>
    <td width="574" valign="top"><p>mannose-1-phosphate guanyltransferase by homology </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>MSH6 </p></td>
    <td width="574" valign="top"><p>DNA mismatch repair protein by homology </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF5671 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF18761 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF5673 </p></td>
    <td width="574" valign="top"><p>Similar to Saccharomyces cerevisiae Rex3p RNA exonuclease (by homology) </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>AKR1 </p></td>
    <td width="574" valign="top"><p>ankyrin repeat-containing protein by homology </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF5678 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF5681 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF13975 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>VMA16 </p></td>
    <td width="574" valign="top"><p>H+-ATPase 23 KD subunit, vacuolar (by homology) </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p><strong>IPF13972 </strong></p></td>
    <td width="574" valign="top"><p><strong>unknown function </strong></p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>RPN1 </p></td>
    <td width="574" valign="top"><p>26S proteasome regulatory subunit (by homology) </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p><strong>IPF9624.5f </strong></p></td>
    <td width="574" valign="top"><p><strong>similar to Saccharomyces cerevisiae Ecm25p involved in cell wall biogenesis and architecture, 5-prime end (by homology) </strong></p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p><strong>IPF9624.3f </strong></p></td>
    <td width="574" valign="top"><p><strong>similar to Saccharomyces cerevisiae Ecm25p involved in cell wall biogenesis and architecture, 3-prime end (by homology) </strong></p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF9626 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>SPE4 </p></td>
    <td width="574" valign="top"><p>spermine synthase (by homology) </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF15844 </p></td>
    <td width="574" valign="top"><p>similar to Saccharomyces cerevisiae Stp2p involved in pre-tRNA splicing </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF19731 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>TCI1 </p></td>
    <td width="574" valign="top"><p>protein phosphatase Two C-Interacting protein (by homology) </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF15968 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
  <tr>
    <td width="76" valign="top"><p>IPF15969 </p></td>
    <td width="574" valign="top"><p>unknown function </p></td>
  </tr>
</table><p>&nbsp;</p>
<p><strong>Signal IP </strong><strong></strong></p>
<p><strong>Prediction: Non-secretory protein </strong><strong></strong></p>
<p>Max cleavage site probability: 0.000 between pos. 28 and 29 </p>
<p><strong>PSORT II </strong><strong></strong></p>
<p>&gt;&gt;&gt; Seems to have <strong>no </strong> N-terminal signal peptide </p>
<p>k = 9/23 </p>
<p>22.2 %: endoplasmic reticulum </p>
<p>22.2 %: nuclear </p>
<p>11.1 %: cytoplasmic </p>
<p>11.1 %: Golgi </p>
<p>11.1 %: vesicles of secretory system </p>
<p>11.1 %: extracellular, including cell wall </p>
<p>&#149;&nbsp; %: cytoskeletal </p>
<p><strong>GPI Prediction server </strong><strong></strong></p>
<p>Potential GPI-modification site was found at 452 </p>
<p><strong>DGPI </strong><strong></strong></p>
<p>Conclusion : </p>
<p>This protein is not GPI-anchored (no signal). </p>
<p>There is a potential cleavage site at 440 (w, w+1, w+2 in red) .</p>
<p>&nbsp;</p>
<p><strong>Representaci </strong><strong>&oacute; </strong><strong>n Gr</strong><strong>&aacute;</strong><strong>fica predicci </strong><strong>&oacute;</strong><strong>n de glicosilaci</strong><strong>&oacute;</strong><strong>n </strong></p>
<span class="Estilo30">&gt;CaSps22|CA0513|IPF13972 SLSSSSSLSSKYVSKISQPTPFVVVVEPLEETPPFSLQQQQQQQQQPPSMDLEMDIPFHC<br> 
RKNEFLIHNQFDLQ <strong>NISN </strong>CEVIIGNIIISEFDYPIITFSNLEKLLG <strong>NLTI </strong>LKSPDTVRID APIMELISGTFKMIELTSLALISFPALKWVNTLHWKILPILSNVHFNNEIKGIDNIIISD TSLTGFSGFLAESIENLDINNNRFLDTIESNVETISGELHIGANSNDVKVSFPKLKQIGL LNINDVEKLNLPELEFIDDSLILNNNNFQQLKFSKLNYIGGTLSLFENLGVNDLEFPNLN <br>
ELGGGLVMI <strong>NNSL </strong>IERINFFPKLKIIGGALELIGNIKEISLKNLKLVKGSAIVKSTSSIF DCKKWSKTEIMLVVRGGRIECTDSNNEKITSRTKEDGTGGELITTPTTTTTTKTNSKSTY SGGRSGGNDDVKMNKETTSSSSGCCGGNVFANGLQHFTLFQFIAILITLVNSFVLPL
</span>
<p>&nbsp; </p><p class="Estilo9">&nbsp;</p>
<p>&nbsp; </p>
<p>&nbsp;</p><p align="left" class="Estilo23">&nbsp;</p></td>
    </tr>
  </table>
  <p>&nbsp;</p>
  <p align="center" class="Estilo4"><span class="Estilo6"><span class="Estilo9">David Jim&eacute;nez-Morales</span><br>
      <span class="Estilo5"> <span class="Estilo14">Departamento de Microbiolog&iacute;a II.<br>
Facultad de Farmacia. <br>
Universidad Complutense de Madrid (UCM) </span><br>
      </span><a href="priv/fecm33davidjmv31.pdf" class="Estilo7">Descargar art&iacute;culo </a></span></p>
  <p align="center" class="Estilo6">&nbsp;</p>
  <p align="center" class="Estilo6">&nbsp;</p>
</div>
</body>
</html>
